modular_skyrat/master_files/code/modules/research/designs/medical_designs.dm

```diff
@@ -0,0 +1,12 @@
+/datum/design/cybernetic_tongue
+	name = "Cybernetic Tongue"
+	desc = "A cybernetic tongue."
+	id = "cybernetic_tongue"
+	build_type = PROTOLATHE | AWAY_LATHE | MECHFAB
+	construction_time = 40
```

Comment:
use seconds defines

Reply:
Hi, none of the other research designs seem to use the `SECONDS`
Can you clarify what you mean?

Reply:
Fixed by commit fca364746b7b7e5de5310bde5155f6b2f0462f91